ULogViewer/Logs/DataSources/StandardOutputLogDataSource.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.Logging;
+
+namespace CarinaStudio.ULogViewer.Logs.DataSources
+{
+	/// <summary>
+	/// Implementation of <see cref="StandardOutputLogDataSource"/>.
```

Comment:
Implementation of <see cref="ILogDataSource"> based-on standard output of process.
-or-
<see cref="ILogDataSource"/> based-on standard output of process.